pages/analyze/[id].tsx

```diff
@@ -107,24 +101,35 @@ const Home: NextPage = () => {
                 </Modal.Footer>
             </Modal> */}
 
-            <main>
-                {/* <div>Match: {state.matchId}</div> */}
-
-                <h1>ANALYSIS BOARD</h1>
+        <main>
+          {/* <div>Match: {state.matchId}</div> */}
 
-                <p>game pgn: {fetchPGN()}</p>
+          <h1>ANALYSIS BOARD</h1>
 
-                <Analysis match_pgn={fetchPGN()} />
+          <p>game pgn: {fetchPGN()}</p>
 
-            </main>
-
-        </div>
-        </div>
-
-    );
+          <Analysis match_pgn={fetchPGN()} />
+        </main>
+      </div>
+    </div>
+  );
 };
 
+export async function getStaticProps() {
+  return {
+    props: {
+      protected: true,
+      premium: true,
+    },
+  };
+}
 
-
+//not sure if this page is being used .. workaround
+export async function getStaticPaths() {
```

Comment:
Not sure if this page is being used, but this was a workaround for an error cause by going to this url given the getstaticprops method defined. Basically a 404 is returned